### DISCUSSION TITLE
Error in freeverb.

### DISCUSSION THREAD

**Discussion by SandeepJagdev** [pschatzmann/arduino-audio-tools#1113] [Q&A] [answered]
I am making reverb padel using arduino-stk . here i found freeverb library but it has no reference code so i followed noise library code but when i call constructor (as same as noise library ) arduino ide giver an error message.
#include"FreeVerb.h"
FreeVerb freeverb ; (Here the  error is)
please help.

**Comment by joba-1** [comment]
Welcome,
and thank you for trusting we can help with such minimal information.
The code you show looks ok, I would at least need the exact error message to see what I can do

**Comment by SandeepJagdev** [comment]
Actually I want to make car dsp product please confirm me that is esp32 based effect for excellent for my product or I use other ic ( such as fv-1 ,adau1701 or tas3204)

**Comment by pschatzmann** [comment, marked as answer]
https://github.com/pschatzmann/arduino-audio-tools/issues/1122
